doc/sphinx/source/input.rst

```diff
@@ -123,28 +123,30 @@ At the moment, cmorize_obs supports Python and NCL scripts.
 
 Cmorization as a fix
 --------------------
-As of early 2020, ESMValTool also provides (limited) support for data in their native format. In this case, the steps needed to reformat the data are executed as datasets fixes during the execution of an ESMValTool recipe, as one of the first preprocessor steps. Compared to the workflow described above, this has the advantage that the user does not need to store a duplicate (cmorized) copy of the data. Instead, the cmorization is performed 'on the fly' when running a recipe. ERA5 is the first dataset for which this 'cmorization on the fly' is supported.
-
-To use this functionality, users need to provide a path for the ``native6`` project data in the :ref:`user configuration file<config-user>`. Then, in the recipe, they can refer to the native6 project, like so:
+ESMValCore also provides support for some datasets in their native
+format. In this case, the steps needed to reformat the data are executed as
+datasets fixes during the execution of an ESMValTool recipe, as one of the first
+preprocessor steps, see :ref:`fixing data <esmvalcore:fixing_data>`.
+Compared to the workflow described above, this has the
+advantage that the user does not need to store a duplicate (cmorized) copy of
+the data. Instead, the cmorization is performed 'on the fly' when running a
+recipe. The native6 project supports data in the format defined in the
+`config-developer file <https://github.com/ESMValGroup/ESMValCore/blob/a9312a7d5be4fa3aac55c0b2ef089c6b4e1a61a9/esmvalcore/config-developer.yml#L191-L201>`_.
```

Comment:
Maybe replace this by a link to the chapter describing the config-developer.yml file? https://docs.esmvaltool.org/projects/esmvalcore/en/latest/quickstart/configure.html#developer-configuration-file

Reply:
Revised.